Update tutorials to latest monthly tag (#607)
substrate-developer-hub/substrate-docs: v3/tutorials/07-add-a-pallet/index.mdx

1. Start the node in development mode by running the following command:

   ```
   ./target/release/node-template --dev
   ```

   The node-template command-line options specify how you want the running node to operate.

>    In this case, the `--dev` option specifies that the node run as a developer node chain specification.

Reviewer: ```suggestion
In this case, the `--dev` option specifies that the node runs in developer mode using the predefined `development` chain specification.
   By default, this option also deletes all active data—such as keys, the blockchain database, and networking information when you stop the node by pressing Control-c.
   Using the `--dev` option ensures that you have a clean working state any time you stop and restart the node.
```

>    By default this option implies that the node will save all active data—such as keys, blockchain database, and networking information—while it is running, then delete the data when you stop the node by pressing Control-c.

Reviewer: ```suggestion
```

>    This also ensures you have a clean working state any time you stop and restart the node.

Reviewer: ```suggestion
```
